src/components/Auth/Registration/Registration.type.ts

```diff
@@ -0,0 +1,8 @@
+export interface IRegistration {
+  username: string,
+  age: string,
+  email: string,
+  password: string,
+  confirmPassword: string,
+  confirmEmail: string,
```

Comment:
type or types we prefer in file names ?

Reply:
For now, type is better because one type with this name exists IUser.type.ts, types/user/IUser.type. I know this can be a little confusing, because some of the types are in a separate directory, second in the directory with the current implementation of the type (Login.type in Login component). I think about what to do with user.type when I will be implementing auth functionality. For now, I prefer to type for file names and move types to the proper location where they are used.

Reply:
So please, rename all files from xxx.type.ts to xxx.types.ts

Reply:
fixed